docs/5-development/09-internationalization.md

```diff
@@ -1 +1,66 @@
-# Internationalization
\ No newline at end of file
+# Internationalization (i18n)
```

Comment:
This is will be displayed in the sidebar `Internationalization (i18n)` - either use the title attribute  to set `Internationalization` or remove `(i18n)`